### ISSUE TITLE
Accessibility: workload, charts, content pages

### ISSUE THREAD

**Issue by lvuch** [rancher/dashboard#4392]
Master: rancher/dashboard#4324

10.
## Dark theme

- Favorite icon contrast issue and aria-role wrong name (should be &quot;role=&#39;button&#39;) ![](RackMultipart20211014-4-19dhorn_html_a3d2a292cecc16dd.png) **image referenced**
- Grey buttons contrast issue

![](RackMultipart20211014-4-19dhorn_html_7815f33cb08600b6.png) **image referenced**

11.
## Shared issues

- This page does not provide additional information to explain the difference in card colors (the line on top that can be blue, yellow or red), only after opening the dropdown ![](RackMultipart20211014-4-19dhorn_html_d03a63b58f3a833a.png) **image referenced** ![](RackMultipart20211014-4-19dhorn_html_cc7186674faa3f9.png) **image referenced**.

We suggest adding an aria-label in each \&lt;div\&gt; to describe the category type for assistive technologies.

12.
## Content pages

- [https://a11y.rancher.space/dashboard/c/local/apps/charts/chart?repo-type=cluster&amp;repo=rancher-partner-charts&amp;chart=confluent-for-kubernetes&amp;version=0.174.2101](https://a11y.rancher.space/dashboard/c/local/apps/charts/chart?repo-type=cluster&amp;repo=rancher-partner-charts&amp;chart=confluent-for-kubernetes&amp;version=0.174.2101)
- [https://a11y.rancher.space/dashboard/c/local/apps/charts/chart?repo-type=cluster&amp;repo=rancher-charts&amp;chart=rancher-backup&amp;version=2.0.1%2Bup2.0.1-rc1](https://a11y.rancher.space/dashboard/c/local/apps/charts/chart?repo-type=cluster&amp;repo=rancher-charts&amp;chart=rancher-backup&amp;version=2.0.1%2Bup2.0.1-rc1)
- [https://a11y.rancher.space/dashboard/c/\_/manager/pages/pod-security-policies](https://a11y.rancher.space/dashboard/c/_/manager/pages/pod-security-policies)

## Shared issues

- Content requires skip links. Skip links are necessary to allow users to navigate through different blocks. [More on this](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&amp;showtechniques=241#qr-navigation-mechanisms-skip).
- **Apply the rule defined here to all other pages with large amount of content displayed**
